diff --git a/node_modules/ssb-serve-blobs/index.js b/node_modules/ssb-serve-blobs/index.js
index dc5fe73..65f8850 100644
--- a/node_modules/ssb-serve-blobs/index.js
+++ b/node_modules/ssb-serve-blobs/index.js
@@ -4,9 +4,11 @@ const FileType = require('file-type');
 const {createUnboxStream} = require('pull-box-stream');
 const BlobsHttp = require('multiblob-http');
 const DEFAULT_PORT = require('./port');
+const {rnBridge} = require("../../ssb/utils");
 
 const zeros = Buffer.alloc(24, 0);
 const FAKE_HOST = 'http://makeurlparseright.com';
+const {app} = rnBridge;
 
 function ServeBlobs(sbot, config) {
   const corsEnabled =
@@ -108,7 +110,16 @@ module.exports = function init(sbot, config) {
       ? config.serveBlobs.port
       : DEFAULT_PORT;
 
-  const server = http.createServer(ServeBlobs(sbot, config)).listen(port);
+  let server;
+  start();
+  function start() {
+    server = http.createServer(ServeBlobs(sbot, config)).listen(port);
+  }
+  app.on('pause', pauseLock => {
+    pauseLock.release();
+    server.close();
+  });
+  app.on('resume', start);
 
   // Ensure that HTTP server is closed when the SSB server closes.
   sbot.close.hook(function (fn, args) {
